FITist_project/user_profiles/urls.py

```diff
@@ -21,4 +22,7 @@
     url(r'^(?P<username>\w+)?/?$',
         get_profile,
         name='user_profile'),
+    url(r'^(?P<username>\w+)/workout_of_the_day$',
```

Comment:
`username` is required in this URL, but not in the one above. Any reason for the inconsistency?